app/src/main/res/values/strings.xml

```diff
@@ -165,11 +165,21 @@
     <string name="no_accounts_found_desc"><![CDATA[ It looks like "<b><font color="#39CBFC"> %1$s </font> </b>" has not been listed on Stax yet or its an invalid entry.<br/>Try searching for something else.]]></string>
     <string name="cant_find_institution">Can\'t find your financial institution?</string>
     <string name="inform_us_here">Inform us here</string>
+
+    <string name="enter_institution_name">Enter Bank, service or account name</string>
+    <string name="enter_shortcode">Enter shortcode</string>
+    <string name="inform_stax_desc">Inform Stax on your financial institution</string>
+    <string name="enter_financial_inst_err">Enter financial institution name</string>
+    <string name="enter_shortcode_err">Enter the accurate shortcode</string>
+    <string name="thank_you">Thank you</string>
+    <string name="service_request_success_desc">Best rest assured the Stax team will review and get right to adding this service soon</string>
```

Comment:
I think it should be `rest assured`. Perhaps we could get some copy help from John or Blessing to make this more formal.